make_timeline.py

```diff
@@ -286,12 +286,7 @@ def plot_multi_line(x, y, z, bins, colors, ax):
     from matplotlib.collections import LineCollection
     from matplotlib.colors import ListedColormap, BoundaryNorm
 
-    # Allow specifying bin centers, not edges
-    if len(bins) == len(colors):
-        bins = np.array(bins, dtype=float)
-        bins = np.concatenate([[z.min() - 1],
-                               (bins[1:] + bins[:-1]) / 2.0,
-                               [z.max() + 1]])
```

Comment:
Gah, no idea how this bug didn't get noticed for 10 years, but I think it is an easy fix here in this code without changing the API:
```
centers = (bins[1:] + bins[:-1]) / 2.0
bins = np.concatenate([centers[0] - 1], centers, [centers[-1] + 1])
```

Reply:
Simplifying the calling code was the intent of the original centered `bins` commit 5b4608e9. Basically `z` is a state-valued array and the bin centers correspond to those state values, which makes sense at least to me.